modules/library/referencing/src/test/java/org/geotools/referencing/operation/transform/ProjectiveTransform2DTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2001-2015, Open Source Geospatial Foundation (OSGeo)
```

Comment:
As this is a new file the header can be simply (C) 2015.

BTW: Thanks for sending in the code contribution agreement - makes this easier.